site/app/views/grading/ElectronicGraderView.php

```diff
@@ -53,7 +53,8 @@ public function statusPage(
         string $section_type,
         int $regrade_requests,
         bool $show_warnings,
-        int $submissions_in_queue
+        int $submissions_in_queue,
+        $sort
```

Comment:
A type hint for the `$sort` parameter should be added here to match the other parameters.